src/qdomyos-zwift.pro

```diff
@@ -9,7 +9,11 @@ qtHaveModule(httpserver) {
     HEADERS += webserverinfosender.h
 }
 
-CONFIG += c++11 console debug app_bundle
```

Comment:
@lifof i had debug on in order to leave the qDebug enabled during the runtime. Are you ok with it?

Reply:
debug disables the compiler optimisations like instead of -O3 it replaces it with an -g

Reply:
qDebug is not impacted by this it's still working on my end.

Reply:
ok thanks

Reply:
The need for the debug flag in the project file, is it because you need to have the name of the file that made the qDebug() followed by debug message ?